networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/libp2p/PeerManagerTest.java

```diff
@@ -40,14 +41,16 @@
 public class PeerManagerTest {
 
   private final ReputationManager reputationManager = mock(ReputationManager.class);
+  private final Gossip gossip = mock(Gossip.class);
```

Comment:
Looks like tests are failing because `Gossip` is final and cannot be mocked.  I suggest injecting the scoring functionality as `Function` like: `new PeerManager(..., (peerId) -> gossip.getGossipScore(peerId))`

Reply:
Yes, indeed, sorry for not noticing it. Thank you for the suggestion, fixed.